docs/getting-started.md

```diff
@@ -0,0 +1,140 @@
+﻿
+# Kubevirtci
+
+Kubevirtci rule is to create a pre provisioned clusters, publish them as a container image,\
+and allow people to run a containerized version of a k8s / open-shift cluster.
+
+Target audience of this project is Developers of kubevirt.
```

Comment:
The target audience should be newcoming developers of kubevirt*ci*.

After reading this document, I should have a basic idea of architecture of kubevirtci providers and be able to create my own one. e.g. a kubernetes provider with IPv6 only.